hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,119 @@
+    - **Platforms** provides a list of platforms your application will support. You can choose to support all platforms, or a subset of them
+    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)) or Uno Platform's MVUX and Feeds
+    - **Projects** gives the ability to add a Server project for APIs and hosting for the WebAssembly project
+    - **Testing** provides Unit Testing and [UI Testing projects](https://github.com/unoplatform/Uno.UITest)
+    - **Features** provides support for WebAssembly PWA and optional VS Code support files
+    - **Extensions** allows to choose for additional [Uno.Extensions](https://github.com/unoplatform/uno.extensions) to kickstart your app faster
+    - **Application** sets the App ID for relevant platforms, used when publishing on various app stores.
+    - **Theme** gives the ability to change between Fluent and Material themes
+
+4. Click the create button
+
+5. Wait for the projects to be created, and their dependencies to be restored
+
+6. A banner at the top of the editor may ask to reload projects, click **Reload projects**:
+    :::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
+
+## Copy your existing WinUI 3 project items
+
+Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy the existing WinUI 3 project items into it.
+
+1. Make sure Visual Studio has your WinUI 3 project open, then copy the following items from the WinUI 3 project to the Uno Platform project:
+    - `MainWindow.xaml`
+    - `MainWindow.xaml.cs`
+
+2. If you're prompted to do so, overwrite any existing files of the same name.
+
+## Deploy your application
+
+1. To run the **WebAssembly** (Wasm) head:
+    - Right click on the `MyApp.Wasm` project, select **Set as startup project**
+    - Press the `MyApp.Wasm` button to deploy the app
+2. To run the ASP.NET Hosted **WebAssembly** (Server) head:
+    - Right click on the `MyApp.Server` project, select **Set as startup project**
+    - Press the `MyApp.Server` button to deploy the app
+3. To debug for **iOS**:
+    - Right click on the `MyApp.Mobile` project, select **Set as startup project**
+    - In the "Debug toolbar" drop-down, select framework `net7.0-ios`:
+
+      :::image type="content" source="images/hello-world/net7-ios-debug.png" alt-text="Visual Studio dropdown to select a target framework to deploy":::
+
+    - Select an active device
+4. To debug the **Android** platform:
+    - Right click on the `MyApp.Mobile` project, select **Set as startup project**
+    - In the **Debug toolbar** drop down, select framework `net7.0-android`
+    - Select an active device in the "Device" sub-menu
+
+Now you're ready to start building your multi-platform application!
```

Comment:
I would suggest adding a 

`## See also`

section at the end to highlight some of the links from the article or some additional resources the readers can jump to next.